### DISCUSSION TITLE
Curso Fundamentos do Blazor Web Assembly:  o código da API não está disponível no GitHub

### DISCUSSION THREAD

**Discussion by engluciana02** [balta-io/forum#579] [Tira Dúvidas] [answered]
No vídeo da Aula Entendendo a API do Projeto fala sobre a execução de uma API que deve ser consumida durante a execução do projeto BaltinhaTest. Mas ao fazer o clone do projeto no Gihub, o diretório encontro-premium-2023-05-04, que deveria conter a API, está vazio. Ela está disponível em outro lugar?

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @engluciana02, como vai?

O repositório desta API está aqui:
https://github.com/balta-io/encontro-premium-2023-05-04

Ficou apenas como referência no repositório do curso mencionado.

**Reply by engluciana02** [reply]
Obrigada.
